controls/def.cf

```diff
@@ -349,7 +349,8 @@ bundle common inventory_control
 # are disabled.
 {
   vars:
-      "lldpctl_exec" string => "/usr/bin/lldpctl";
```

Comment:
I think other platforms may have `/usr/local/bin/lldpctl` and it will Just Work there too.